fmriprep/workflows/bold/base.py

```diff
@@ -67,500 +69,311 @@ def init_func_preproc_wf(bold_file, has_fieldmap=False):
 
             from fmriprep.workflows.tests import mock_config
             from fmriprep import config
-            from fmriprep.workflows.bold.base import init_func_preproc_wf
+            from fmriprep.workflows.bold.base import init_bold_wf
             with mock_config():
                 bold_file = config.execution.bids_dir / "sub-01" / "func" \
                     / "sub-01_task-mixedgamblestask_run-01_bold.nii.gz"
-                wf = init_func_preproc_wf(str(bold_file))
+                wf = init_bold_wf(
+                    bold_series=[str(bold_file)],
+                )
 
     Parameters
     ----------
-    bold_file
-        Path to NIfTI file (single echo) or list of paths to NIfTI files (multi-echo)
-    has_fieldmap : :obj:`bool`
-        Signals the workflow to use inputnode fieldmap files
+    bold_series
+        List of paths to NIfTI files.
+    precomputed
+        Dictionary containing precomputed derivatives to reuse, if possible.
+    fieldmap_id
+        ID of the fieldmap to use to correct this BOLD series. If :obj:`None`,
+        no correction will be applied.
 
     Inputs
     ------
-    bold_file
-        BOLD series NIfTI file
     t1w_preproc
         Bias-corrected structural template image
     t1w_mask
         Mask of the skull-stripped template image
     t1w_dseg
         Segmentation of preprocessed structural image, including
         gray-matter (GM), white-matter (WM) and cerebrospinal fluid (CSF)
-    t1w_aseg
-        Segmentation of structural image, done with FreeSurfer.
-    t1w_aparc
-        Parcellation of structural image, done with FreeSurfer.
```

Comment:
Do we care about resampling aseg/aparc into template spaces?

Reply:
this seems like a good thing to push to `fmripost`